solidstart-tanstackquery-tailwind-modules/src/services/get-pull-request.ts

```diff
@@ -59,11 +80,31 @@ function parsePullRequests(connection?: any) {
   return { pullRequests, totalCount, pageInfo };
 }
 
+function parseLabels(labels: {
+  totalCount: number;
+  nodes: LabelProps[];
```

Comment:
Why `LabelProps[]` here instead of `Label[]`?

Reply:
Yeah this is one of the things that we got from the SolidJS showcase, probably we should solve it in the cleanup ticket because they are a lot all over the showcase.